src/main/java/com/minecolonies/coremod/entity/pathfinding/PathNavigate.java

```diff
@@ -303,31 +289,31 @@ else if (entity.isInWater())
 
                 this.getPath().setCurrentPathIndex(oldIndex);
 
-                Vec3d Vec3d = this.getPath().getPosition(this.entity);
+                Vec3d vec3d = this.getPath().getPosition(this.entity);
 
-                if (Vec3d.squareDistanceTo(new Vec3d(entity.posX, Vec3d.yCoord, entity.posZ)) < 0.1
-                      && Math.abs(entity.posY - Vec3d.yCoord) < 0.5)
+                if (vec3d.squareDistanceTo(new Vec3d(entity.posX, vec3d.yCoord, entity.posZ)) < 0.1
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.1 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)